content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,114 @@ title: What is an **attribute**?
 related: manage-your-attributes, what-is-an-attribute-group
 ---
 
-# Definition of an attribute
-
-An attribute is a product’s characteristic. Each product is composed of different attributes.
-
-Depending on your Akeneo Edition version, you can have up to 15 attribute types:
-- **identifier**: a unique code that identifies your product
-- **text** and **text area**: used for text data. A **text** attribute is **limited to 255 characters**, if you need more text data, use the **text area** attribute type.
-- **simple** or **multi select**
-- **reference data** (simple and multi select)
-- **boolean** (yes/no)
-- **date**
-- **number**
-- **price**
-- **metric** (with automatic conversions): this attribute type allows you to **automatically convert metric values into others to fit your export needs**. See the [Manage your Channels](manage-your-channels.html#create-a-channel) article.
-- **media file**: a simple file (pdf file for instance) or an image
-- **[asset collection](work-with-assets.html)**: digital resources like a video, picture, pdf file... _(Entreprise Edition only)_
-
-Attributes are gathered into [attribute groups](what-is-an-attribute-group.html) in Akeneo.
+# Introduction
+
+An attribute is a product’s characteristic. A product is usually composed of several attributes: an identification number, a name, a description, a price, a color... Depending on your Akeneo Edition version (Community or Enterprise), you can choose amongst **17 attribute types**. 
+
+In Akeneo, attributes are gathered into **families**, so all products belonging to the same family share the same attributes. If you are not familiar with the concept of families, check our [What is family?](what-is-a-family.html) article.
+
+You can easily create attribute groups to gather your attributes and better manage them. Read the following article to get more information about attribute groups: [What is an attribute group?](what-is-an-attribute-group.html).
```

Comment:
Pour bien différencier de la famille, je mettrai plutôt "to gather your attributes and ease their enrichment in the product forrm".